snapcraft.yaml

```diff
@@ -18,39 +18,38 @@ slots:
 
 apps:
   gnome-2048:
-    extensions: [gnome-3-38]
+    extensions: [gnome]
     command: usr/bin/gnome-2048
     desktop: usr/share/applications/org.gnome.TwentyFortyEight.desktop
     common-id: org.gnome.TwentyFortyEight
+    environment:
+      DISABLE_WAYLAND: 'true'
```

Comment:
Oof, disable wayland just forever? maybe add a comment guiding us when we can remove this.

Reply:
Added a note specifying to try again when it gets ported from Clutter to Gtk4.